src/graph.py

```diff
@@ -23,11 +23,14 @@ def has_edge(self, u, v):
                 result = True
         return result
 
-    def get_spanning_tree(self, start):
+    def get_spanning_tree(self, start, seed = None):
         spanning_tree = Graph(self.num_nodes)
         stack = []
         visited = [False]*self.num_nodes
 
+        if seed:
```

Comment:
nice

Reply:
You got a lot of issues. Call me the truth >:)